jbi/actions/default.py

```diff
@@ -96,8 +98,23 @@ def __call__(self, context: ActionContext) -> ActionResult:
 
         responses = tuple()  # type: ignore
 
+        incomplete = False
+
         for step in self.steps[context.operation]:
-            context, step_responses = step(context=context, **self.parameters)
+            try:
+                context, step_responses = step(context=context, **self.parameters)
+            except Exception:
+                if len(responses) > 0:
+                    # Count the number of workflows that produced at least request,
+                    # but could not complete entirely successfully.
+                    statsd.incr("jbi.bugzilla.aborted.count")
```

Comment:
Should we add the name of the workflow somewhere in the name of this stat? Might be helpful to see which workflows are aborted or finish as incomplete the most.